frontend/install.rst

```diff
@@ -79,8 +77,8 @@ directory:
 .. code-block:: bash
 
     $ cd irma-frontend/web
-    $ npm install
-    $ node_modules/.bin/bower install
+    $ sudo npm install
```

Comment:
@ch0k0bn, normally there is no need for sudo there. But due to directory owner, you should be doing this command `npm install` and other with `deploy` user.